### DISCUSSION TITLE
Azure Stack Hub Support

### DISCUSSION THREAD

**Discussion by casperwwt** [pulumi/pulumi#8895] [Ideas]
With Azure Stack Hub being (almost) consistent with Azure Public, can we get a way to target Stack Hub? This does not seem possible at this stage.

**Comment by mikhailshilkov** [comment]
Do you mean we should extend our Azure Native provider to support Stack Hub?

**Reply by casperwwt** [reply]
Yes or a separate provider. There doesn't seem to be a way to target Stack Hub, even though the APIs are consistent, versioning aside.

**Comment by aureq** [comment]
We have opened a separate feature request to assist with the tracking of this conversation. Please subscribe and upvote https://github.com/pulumi/pulumi/issues/12645
